apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/nfts/components/table.tsx

```diff
@@ -61,11 +60,7 @@ export const NFTGetAllTable: React.FC<ContractOverviewNFTGetAllProps> = ({
       {
         Header: "Token Id",
         accessor: (row) => row.id?.toString(),
-        Cell: (cell: CellProps<NFT, string>) => (
-          <Text size="body.md" fontFamily="mono">
-            {cell.value}
-          </Text>
-        ),
+        Cell: (cell: CellProps<NFT, string>) => <p>{cell.value}</p>,
```

Comment:
Missing `font-mono`

Reply:
Can you tell me why do we want this mono font in some specific place? I saw little to no visual difference so I removed it intentionally